build.gradle

```diff
@@ -2,6 +2,7 @@ buildscript {
     ext.gradle_version = '7.4.2'
     ext.kotlin_version = '1.8.10'
     ext.navigation_version = '2.5.3'
+    ext.serialization_json_version = '1.5.0' // TODO: Temporary support for kDrive, remove this when it's fixed in :Core
```

Comment:
<img width="645" alt="image" src="https://user-images.githubusercontent.com/1788629/224919345-3dd79f69-294d-44f2-ae99-568d69f943f6.png">
What is `kotlinx_json_version` ?

Reply:
Yes indeed, it is an oversight